scripts/apel-build.sh

```diff
@@ -111,6 +111,12 @@ mkdir -p "$TEMP_DIR_FOR_SERVER/var/run/apel"
 # apel-lib
 mkdir -p "$TEMP_DIR_FOR_LIB/$PYTHON_ROOT_DIR/$LIB_EXTENSION/apel/"
 
+# Ensure systemd's tmpfiles creates the /var/run/apel folder on every reboot
+mkdir -p "$TEMP_DIR_FOR_SERVER/etc/tmpfiles.d"
+cat > "$TEMP_DIR_FOR_SERVER/etc/tmpfiles.d/run-apel.conf" <<EOF
```

Comment:
It took me a while to parse what was happening here, it's a use of `cat` I hadn't come across before. Maybe an additional comment explaining what it is doing would be useful.